site_scons/prereq_tools/base.py

```diff
@@ -1590,12 +1586,10 @@
 
         # Make sure CheckProg() looks in the component's bin/ dir
         if not self.use_installed and not self.component_prefix == "/usr":
-            env.AppendENVPath('PATH', os.path.join(self.component_prefix,
-                                                   'bin'))
+            env.AppendENVPath('PATH', os.path.join(self.component_prefix, 'bin')
```

Comment:
(pylint-syntax-error) Generator expression must be parenthesized (<unknown>, line 1589)